Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs

```diff
@@ -0,0 +1,1313 @@
+--                      , primaryText: "Expert Driving"
+--                      , subText: "235"
+--                       },
+--                       {badgeImage: "ny_ic_navigator_badge,https://assets.juspay.in/nammayatri/images/driver/ny_ic_navigator_badge.png"
+--                      , primaryText: "Navigator"
+--                      , subText: "235"
+--                       },
+--                       {badgeImage: "ny_ic_ontime_badge,https://assets.juspay.in/nammayatri/images/driver/ny_ic_ontime_badge.png"
+--                      , primaryText: "On Time"
+--                      , subText: "235"
+--                       },
+--                       {badgeImage: "ny_ic_polite_driver_badge,https://assets.juspay.in/nammayatri/images/driver/ny_ic_polite_driver_badge.png"
+--                      , primaryText: "Professional"
+--                      , subText: "235"
+--                       }
+--                       ]
+  
+transactionView :: forall w . (Action -> Effect Unit) -> ST.DriverEarningsScreenState -> PrestoDOM (Effect Unit) w
+transactionView push state = 
+  linearLayout
+    [ width MATCH_PARENT
+    , height WRAP_CONTENT
+    , orientation VERTICAL
+    , margin $ MarginTop 20
+    ] [ linearLayout 
+          [ width MATCH_PARENT
+          , height WRAP_CONTENT
+          , orientation HORIZONTAL
+          , margin $ Margin 0 0 8 13
+          ] [ textView $ [
+              text $ getString TRANSACTION_HISTORY
+            , weight 1.0
+            , color Color.black800     
+            ] <> FontStyle.h2 TypoGraphy      
+            , calendarView push state    
+          ]
+          , historyView push state
+    ]
+
+calendarView :: forall w . (Action -> Effect Unit) -> ST.DriverEarningsScreenState -> PrestoDOM (Effect Unit) w
+calendarView push state = 
+  let selectedDate = state.props.date
+      dateToShow = if DS.null selectedDate || convertUTCtoISC selectedDate "YYYY-MM-DD" == getcurrentdate "" 
+                   then getString TODAY 
+                   else convertUTCtoISC selectedDate "Do MMM"
+  in
+  linearLayout
+    [ width WRAP_CONTENT
+    , height WRAP_CONTENT
+    , cornerRadius 100.0
+    , background Color.white900
+    , gravity CENTER_VERTICAL
+    , stroke $ "1," <> Color.grey900
+    , padding $ Padding 12 4 12 4
+    , onClick push $ const $ ShowCalendarPopup
+    ][ imageView [
+        imageWithFallback $ fetchImage FF_ASSET "ny_ic_calendar_unfilled_blue,"
+      , height $ V 16
+      , width $ V 16
+      , margin $ MarginRight 4
+      ]
+      , textView $ [
+        text dateToShow
+      , color Color.black700
+      ] <> FontStyle.tags TypoGraphy
+      , imageView [
+        imageWithFallback $ fetchImage FF_ASSET if state.props.calendarState.calendarPopup 
+                          then "ny_ic_chevron_up,https://assets.juspay.in/beckn/nammayatri/nammayatricommon/images/ny_ic_chevron_up.png"
```

Comment:
replace the link with just the image name as fetchImage just takes image name as parameter.